submissions/QuickPerformance/QuickPerformance.md

```diff
@@ -0,0 +1,113 @@
+
+The easiest solution to a friend not being able to catch up is simply pausing the game. That can be done by pressing `shift+space`. This is however often only temporary, as the game will still simulate after you unpause the game, so your friend's computer may not be able to keep up.
+
+If you have a lot of pollution in places and you are further into the game where biters are just an expansion nuisance, you can run this command:
+
+```lua
+/c for _, surface in pairs(game.surfaces) do
+  surface.clear_pollution()
+end
+game.map_settings.pollution.enabled = false
+```
+
+For every surface in the game, it will remove all pollution and after that, it will disable pollution creation, which can improve performance quite a lot in some cases.
+
+If you however believe that pollution is not the right thing for you and instead you have large amounts of biters that are just idle, you can run this command instead:
+
+```lua
+/c game.forces["enemy"].kill_all_units()
+```
+
+It will kill all biters on the `"enemy"` force (which without any modifications will be biters). It will however **only remove mobile units, such as biters and spitters, not spawners or turrets**. If you would like to remove those too, you can use this command instead:
+
+```lua
+/c local surface=game.player.surface
+for key, entity in pairs(surface.find_entities_filtered({force="enemy"})) do
+	entity.destroy()
+end
+```
+
+If you *still* feel that biters are just annoying and your base always just deals with them without any input, you can use this command instead:
+
+```lua
+/c local surface = game.player.surface
+local mgs = surface.map_gen_settings
+mgs.autoplace_controls["enemy-base"].size = "none"
+surface.map_gen_settings = mgs
+```
+
+It will disable biters from generating **only on your surface**, which by default is *Nauvis*, however it won't affect current biters (you will need to use previous commands for that).
+
+If none of these work, the easiest resort to go to is reducing the game speed. Everything will run slower, but you will be able to play with your friends without them getting kicked off:
+
+```lua
+/c game.speed = 0.8
+```
+This command slows down the game by changing it's speed (and thereby decreasing UPS). The default is 1, but you can go as low as 0.01, where pretty much nothing is happening. The best idea is to start going down from 1 in increments of 0.1 or 0.05, such as 0.9, 0.8, 0.7 etc. This command will globally reduce the game's speed, so there will be less updates per second to calculate. The UPS number in the corner of your screen will be multiplied from the base by this constant, in the way $1*60=60$, $0.8*60=48*$, $0.5*60=30$. 
+
+All of these have been sourced from the [Commands wiki page](https://wiki.factorio.com/Console) if you want to see more Lua magic. To run any command, simply copy it, open your chatbar and paste it in.
+
+### Optimization by Design
```

Comment:
For the final conclusion, maybe change the final two paragraphs around to make it more conslusiony